app/gui/SettingsView.qml

```diff
@@ -759,9 +772,25 @@ Flickable {
                         activated(currentIndex)
                     }
 
+                    // Video Super-Resolution does not work in exclusive full screen, so auto switch do borderless window
+                    // [TODO] This may change according to what AMD and Intel will implement, if they can allow video enhancement in fullscreen
```

Comment:
I checked which engine is used while I select "exclusive fullscreen", it is D3D11VA, so I will keep the current UI behavior for now. I guess the limitation comes from NVIDIA implementation, it is maybe related to WDM somehow, I am asking NVIDIA to know more about it:
https://www.nvidia.com/en-us/geforce/forums/game-ready-drivers/13/537893/rtx-video-enhancement-not-working-in-exclusive-ful/

Reply:
That's strange because we aren't even using true exclusive fullscreen. SDL is setting the resolution for us, then we create a regular windowed swapchain.

Reply:
Done
I just updated to Nvidia 551.61 and now exclusive fullscreen works with VSR.
I also tested an Arc a380 and it works in all windows modes as well.
As for AMD, I will receive a RX 7600 soon, I will tested later once I implement its VSR solution.
Therefore, I can remove some lines in SettingsView.qml to always allow VSR regardless the Window mode.